bazel/linux/README.md

```diff
@@ -2,33 +2,41 @@
 Bazel hooks for compiling and testing linux kernel modules against a specific kernel version.
 
 # Main rules made available
-`kernel_version` defines a specific kernel version to compile and test modules against.
+`kernel_tree_version`* defines a specific kernel tree to compile and test modules against.
```

Comment:
What is the '*' after kernel_tree_version? not sure what it achieves. Italic? but there is no closing *. Bold? same.

Reply:
My intention was to use it as as a reference to the footnote (line 20). Does markdown have "native" footnotes?